news/2018-04-03-spring-2018-fanart-contest.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Spring 2018 Fanart Contest"
+date: 2018-04-03 07:00:00 +0000
+---
+
+We're ready to appreciate the blooming talents of our amazing artists once again, in 2018's first Fan Art contest, featuring spring! Like to draw and want some free osu!supporter? Check this out!
```

Comment:
`Fan Art` -> `Fanart`?
the title and this occurrence don't match, I guess... then again, shouldn't it be `fanart contest` or `Fanart Contest`?

Reply:
good pick there actually, hmmm. old articles use fanart so I guess i'll go with that to keep consistent with how things historically have been